builtins/amp-img.js

```diff
@@ -41,6 +47,17 @@ export class AmpImg extends BaseElement {
   }
 
   /** @override */
+  attributeChangedCallback(name, unusedOldValue, unusedNewValue) {
```

Comment:
So, does `attributeChangedCallback` already assumes mutate context? If so, please document this at the API level.